src/cargo/ops/cargo_rustc/compilation.rs

```diff
@@ -23,6 +23,7 @@ pub struct Compilation {
     ///
     /// This is currently used to drive some entries which are added to the
     /// LD_LIBRARY_PATH as appropriate.
+    // TODO: deprecated, remove
```

Comment:
Is this TODO correct, @alexcrichton?  I.e. should I PR the removal of `Compilation.native_dirs`, or PR the removal of the TODO?

Reply:
Oh dear that's a good question, this PR is... quite old though! I vaguely remember placing this on a few things throughout Cargo I've wished I deleted at the time and then never got to, but removing this would be good if it's not actually used in Cargo today! (or if it's actually used just remove the comment)